app/src/main/res/layout-land/faq_single_activity.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.story.StoryViewModel" />
```

Comment:
Why we need `StoryViewModel` in faq?

Reply:
Removed it in both files in latest commit.

Reply:
Thanks, Good to go.